src/doc/rustc/src/check-cfg.md

```diff
@@ -14,6 +14,9 @@ whether to check conditions and how to check them.
 > **Note:** No implicit expectation is added when using `--cfg`. Users are expected to
 pass all expected names and values using the _check cfg specification_.
 
+For interacing with this through Cargo,
+see [Cargo Specifics - Checking Conditional Configurations](check-cfg/cargo-specifics.md).
```

Comment:
Isn't a bit redundant to call out the "Checking Conditional Configurations" part inside the check-cfg page?

Why not?
```suggestion
For interacing with this through Cargo,
see the [Cargo Specifics](check-cfg/cargo-specifics.md) subpage.
```

Reply:
Tweaked the wording